### PR TITLE
made response nicer

### LINKED ISSUE #304: Sentiment Analysis output is boring
It simply writes the text "Positive" or "Negative" in a not-very-obvious way, with no notion of how certain the model is despite that information existing in the response.

```
{"label":"1","logits":[5.217404842376709,-5.244380474090576],"probs":[0.9999713897705078,2.8608292268472724e-05],"slug":"MTM3MTc0MA=="}
```

### PR DESCRIPTION
**Pull request by jonborchardt**
fixes #304

image
image
image

### CODE REVIEW

**Comment by jonborchardt** [comment]
sure, ill add percent.

**Comment by matt-gardner** [comment]
I'm not sure that we should use the word "we" for model outputs.  This is a model, not us.  We're not endorsing the model necessarily, and in most cases it's not a model that we developed.  I'd recommend changing all uses of "we" in what I see here to "the model", or something similar.

**Comment by jonborchardt** [comment]
will do

**Comment by jonborchardt** [comment]
image
image
image
image

File: ui/src/components/demos/SentimentAnalysis.js [hunk 1]
```diff
@@ -120,12 +121,27 @@ const Attacks = ({attackData, attackModel, requestData}) => {
...
 }
 
+const textFromProbability = (prob) => {
+  const probPosStr = Number(prob).toLocaleString(undefined, {style: 'percent', minimumFractionDigits: 1});
+  const probNegStr = Number(1-prob).toLocaleString(undefined, {style: 'percent', minimumFractionDigits: 1});
+  if (prob < 0.2) {
+      return <div>The model is quite sure the sentence is <Negative>Negative</Negative>. ({probNegStr})</div>;
+  } else if (prob < 0.5) {
+      return <div>The model thinks the sentence is <Negative>Negative</Negative>. ({probNegStr})</div>;
+  } if (prob == 0.5) {
```
Comment (matt-gardner) [inline ui/src/components/demos/SentimentAnalysis.js:131]:
```suggestion
  } else if (prob == 0.5) {
```

**Pull request merged**